SPEC.md

```diff
@@ -7367,6 +7368,93 @@ Example output:
 </p>
 </details>
 
+### ✨ `join_paths`
+
+```
+File join_paths(File|String, String)
+File join_paths(File, Array[String]+)
+File join_paths(Array[String]+)
+```
+
+Joins together two or more paths into an absolute path in the host filesystem.
+
+There are three variants of this function:
+
+1. `File join_paths(File|String, String)`: Joins together exactly two paths. The first path may be either absolute or relative and must specify a directory; the second path is relative to the first path and may specify a file or directory.
```

Comment:
If File is coercible to a String, is this needed, or just helpful to imply?

Reply:
Good point. `String` is coercible to `File` so we only need `File`.